Browser.py

```diff
@@ -106,6 +109,21 @@ def login(self, username: str, password: str, refreshLock) -> bool:
             if resAccessToken.status_code == 200:
                 #self.maintainSession()
                 self.__dumpCookies()
+                res = (self.client.cookies.get_dict()["id_hint"])
+                urlDecoded = urllib.parse.unquote(res)
+                newName = re.search(r"&summoner=(.*)&region", str(urlDecoded))
+                region = re.search(r"&region=(.*)&tag", str(urlDecoded))
+                if newName.group(1):
+                    if region.group(1):
```

Comment:
You can combine these into 1 if statement
```suggestion
                if newName.group(1) AND region.group(1):
```